src/pybind/mgr/dashboard/frontend/src/testing/unit-test-helper.ts

```diff
@@ -31,7 +31,7 @@ import {
 } from '~/app/shared/models/prometheus-alerts';
 
 export function configureTestBed(configuration: any, entryComponents?: any) {
-  configureTestSuite(() => {
+  configureSuite(() => {
```

Comment:
Is this still required? I read 'somewhere' that Angular fixed this issue internally, so maybe we can disable this in a couple a files and see if tests are still sped up...

BTW, in a follow up work we should consider adding [Spectator](https://github.com/ngneat/spectator) for [simplifying our component testing](https://testing-angular.com/testing-components-with-spectator/#testing-components-with-spectator). Also, [Cypress can be used for component testing](https://docs.cypress.io/guides/component-testing/testing-angular#Selecting-the-StepperComponent).

Reply:
> Is this still required? I read 'somewhere' that Angular fixed this issue internally,

I'll test it with and without these changes and compare its performance. I read that as well they fixed the issue but I've read more threads that the test isn't sped up even after the jest's internal fix.



> in a follow up work we should consider adding [Spectator](https://github.com/ngneat/spectator) for

Yup, I tried to introduce the spectator here but it looked like a lot of work so I didn't went for it here. A follow up makes sense.

Reply:
I compared the performance.

With the current implementation, the time it took is about `2min and couple of seconds` to finish. But by disabling it, it took about `4min` to complete.

So I am going to keep this implementation and maybe try to improve it later on.